.taskcat.yml

```diff
@@ -0,0 +1,202 @@
+    # These settings should be left alone unless you know what you're doing.
+    # These parameters identify the S3 buckets where the CloudForomation
+    # resources can be found.
+    XrdS3BucketName: $[taskcat_autobucket]
+    # The key prefix needs to match <project name>/ (where project name is
+    # configured below) to work with taskcat's autosync.
+    XrdS3KeyPrefix: xrd-eks/
+    XrdS3BucketRegion: $[taskcat_current_region]
+    # Where the xrd-overlay-overlay Helm chart can be found.
+    HelmRepository: https://ios-xr.github.io/xrd-eks/
+    # =========================================================================
+
+    # The rest of the settings below here can freely be changed.
+    AvailabilityZones: $[taskcat_genaz_2] # "us-east-2a,us-east-2b"
+
+    # EKS Cluster settings.
+    ClusterName: "xrd-cluster"
+    EKSClusterName: "xrd-cluster"
+    # Remote access to the EKS cluster.
+    RemoteAccessCIDR: "0.0.0.0/0"
+    EKSPublicAccessEndpoint: Enabled
+
+    # XRd login details.
+    # These must be set before running XRd.
+    #XrdRootUserName: changeme
+    #XrdRootPassword: changeme
+
+project:
+  # Sets key prefix for all files within the S3 bucket.
+  # This must match the `XrdS3KeyPrefix` above and must be kept in sync with
+  # the publish-s3-bucket and create-stack scripts in this repository.
+  name: 'xrd-eks'
+  # Using pinned versions of the submodules, including built resources,
+  # so don't build them here.
+  build_submodules: false
+  # No lambdas to package, so can set this to false.
+  # This also removes the dependency on docker.
+  package_lambda: false
+
+tests:
+  # Create an AMI.
+  xrd-eks-ami:
+    template: cf-templates/xrd-eks-ami-cf.yaml
+
+  # Create a VPC.
+  xrd-vpc:
+    template: cf-templates/xrd-vpc-cf.yaml
+
+  # Create an EKS cluster in an existing VPC.
+  xrd-eks-existing-vpc:
+    template: cf-templates/xrd-eks-existing-vpc-cf.yaml
+
+  # Create a new VPC and an EKS cluster inside it.
+  xrd-eks-new-vpc:
+    template: cf-templates/xrd-eks-new-vpc-cf.yaml
+
+  # Create a worker node with four interfaces.
+  xrd-eks-node:
+    template: cf-templates/xrd-eks-node-cf.yaml
+    parameters:
+      WorkerNodeName: "standalone-worker"
+      NodeInstanceType: "m5n.2xlarge"
+      # Primary IP for eth0 use for cluster networking.
+      NodeIP: "10.0.101.11"
+
+      # m5n.2xlarge instances are limited to 3 XRd interfaces - the instance
+      # type supports maximum of four interfaces, one of which is used by the
+      # host rather than XRd.
+      XrdInterface1IP: "10.0.1.10"
+      XrdInterface2IP: "10.0.2.10"
+      XrdInterface3IP: "10.0.3.10"
+
+      XrdInterface1SubnetTagValue: 1
+      XrdInterface2SubnetTagValue: 2
+      XrdInterface3SubnetTagValue: 3
+
+  # Launch a worker node and run a single XRd vRouter instance on it.
+  xrd-singleton-vrouter:
+    template: cf-templates/xrd-singleton-existing-eks-cf.yaml
+
+  # Launch a worker node suitable for a single XRd node launched by
```

Comment:
I don't think this comment is very clear - the point is `xrd-singleton-vrouter-helm` doesn't launch anything.